TYPO3.Neos/Resources/Private/TypoScript/Prototypes/Page.ts2

```diff
@@ -58,28 +58,31 @@ prototype(TYPO3.Neos:Page) < prototype(TYPO3.TypoScript:Http.Message) {
 			templatePath = 'resource://TYPO3.Neos/Private/Templates/TypoScriptObjects/NeosBackendHeaderData.html'
 			node = ${documentNode}
 
-			@cache {
-				mode = 'uncached'
-				context {
-					1 = 'documentNode'
+			@if.onlyRenderWhenNotInLiveWorkspace = ${documentNode.context.inBackend}
+		}
+
+		neosBackendUncachedHeaderData = TYPO3.TypoScript:Value {
+			value = TYPO3.TypoScript:Template {
+				@position = 'end 10001'
+				templatePath = 'resource://TYPO3.Neos/Private/Templates/TypoScriptObjects/NeosBackendUncachedHeaderData.html'
+				account = ${Security.getAccount()}
+
+				@cache {
+					mode = 'uncached'
+					context {
+						1 = 'documentNode'
+					}
 				}
 			}
 
 			@if.onlyRenderWhenNotInLiveWorkspace = ${documentNode.context.inBackend}
 		}
 
 		neosBackendEndpoints = TYPO3.TypoScript:Template {
-			@position = 'end 10001'
+			@position = 'end 10002'
 			templatePath = 'resource://TYPO3.Neos/Private/Templates/TypoScriptObjects/NeosBackendEndpoints.html'
 			node = ${documentNode}
-			account = ${Security.getAccount()}
 
-			@cache {
-				mode = 'uncached'
-				context {
-					1 = 'documentNode'
-				}
-			}
 			@if.onlyRenderWhenNotInLiveWorkspace = ${documentNode.context.inBackend}
```

Comment:
Hey sorry, but this won't work for several reasons.

The view helpers used in the template cannot work when cached, e.g. `neos:backend.configurationCacheVersion`, `neos:backend.interfaceLanguage` and `neos:backend.xliffCacheVersion`. It would be possible to cache if those could be retrieved in TypoScript so they could be added to the cache tag.

Also there are a couple of context variables used, which would need to be added to the cache tag to work. Currently `node.context.currentSite.assetCollection` and `node.context.currentSiteNode` are used.

Btw. a bit of deja vu feeling here 😃